features/_debug/file.include/usr/sbin/setup-debugbox-ssh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+
+set -Eeufo pipefail
+
+IMAGE="debugbox"
+REGISTRY=""
+CMD='/usr/sbin/sshd -D'
+CONTAINER_ARCHIVE=""
+
+if [ -f /etc/debugbox.conf ]; then
+    source /etc/debugbox.conf
+fi
+
+if [[ -z "${REGISTRY}" ]]; then
+    if [[ -z "${CONTAINER_ARCHIVE}" ]]; then
+        build_dir=/var/tmp/debugbox/ssh
+        mkdir -p ${build_dir}
+
+        cd $build_dir
+
+        cat <<EOF > Dockerfile
+FROM debian:testing-slim
```

Comment:
This is an issue. Adding debian:testing-slim to the garden linux image itself would include libdb, and potentially other unpatched binaries.